runtime/syntax/log.yaml

```diff
@@ -0,0 +1,94 @@
+filetype: log
+
+detect:
+    filename: "(\\.log|log\\.txt)"
```

Comment:
Better to add a `$` at the end `(\\.log|log\\.txt)$`, otherwise `test.logging` will fit too, am I right?

Reply:
Adjusted